<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pmcaconf_mainwin</class>
 <widget class="QMainWindow" name="pmcaconf_mainwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMCA set configuration tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/logo/logo.png</normaloff>:/logo/logo.png</iconset>
  </property>
  <property name="locale">
   <locale language="Hungarian" country="Hungary"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>41</y>
      <width>171</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Groups</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_parts">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>171</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>16</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parts</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>79</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Attachable groups</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_availableGroups">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>131</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Material type</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>97</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid color</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>97</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Material Editor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>List ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_list">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Root directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imageBut">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_configuration"/>
    <addaction name="actionSave_configuration"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_part"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_configuration">
   <property name="text">
    <string>Open configuration</string>
   </property>
  </action>
  <action name="actionSave_configuration">
   <property name="text">
    <string>Save configuration</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_part">
   <property name="text">
    <string>Add part</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>pmcaconf_mainwin</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_configuration</sender>
   <signal>triggered()</signal>
   <receiver>pmcaconf_mainwin</receiver>
   <slot>openConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>pmcaconf_mainwin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_part</sender>
   <signal>triggered()</signal>
   <receiver>pmcaconf_mainwin</receiver>
   <slot>addNewPart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_parts</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pmcaconf_mainwin</receiver>
   <slot>changePart(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pmcaconf_mainwin</receiver>
   <slot>switchPartGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAbout()</slot>
  <slot>openConfig()</slot>
  <slot>addNewPart()</slot>
  <slot>changePart(QModelIndex)</slot>
  <slot>switchPartGroup()</slot>
 </slots>
</ui>
